python/ray/data/read_api.py

```diff
@@ -1519,13 +1734,24 @@ def from_modin(df: "modin.DataFrame") -> MaterializedDataset:
 def from_pandas(
     dfs: Union["pandas.DataFrame", List["pandas.DataFrame"]]
 ) -> MaterializedDataset:
-    """Create a dataset from a list of Pandas dataframes.
+    """Create a :class:`~ray.data.Dataset` from a list of pandas dataframes.
+
+    Examples:
+        >>> import pandas as pd
+        >>> import ray
+        >>> df = pd.DataFrame({"a": [1, 2, 3], "b": [4, 5, 6]})
+        >>> ray.data.from_pandas(df)
+        MaterializedDataset(num_blocks=1, num_rows=3, schema={a: int64, b: int64})
+
+        >>> # Create a Ray Dataset from a list of Pandas DataFrames.
+        >>> ray.data.from_pandas([df, df])
+        MaterializedDataset(num_blocks=2, num_rows=6, schema={a: int64, b: int64})
 
     Args:
-        dfs: A Pandas dataframe or a list of Pandas dataframes.
+        dfs: A pandas dataframe or a list of Pandas dataframes.
```

Comment:
```suggestion
        dfs: A pandas dataframe or a list of pandas dataframes.
```

Reply:
I thought Pandas was capitalized, but I see that you're deliberately using lower-case.